packages/ws-server/tests/acceptance/subscribeNewHeads.spec.ts

```diff
@@ -32,6 +32,7 @@ import { ethers } from 'ethers';
 import WebSocket from 'ws';
 
 import { WsTestHelper } from '../helper';
+import { ConfigKey } from '../../../config-service/src/services/globalConfig';
```

Comment:
I see many changes due to only `import`s, I guess this is a leftover from a previous commit. If there is not actual change, we should revert these changes for easier review.